### DISCUSSION TITLE
Website's copyright year variable isn't correct

### DISCUSSION THREAD

**Discussion by rpaladin** [twbs/bootstrap#38068] [General]
Asking here because I'm not sure if this belongs under repository Issues. The code for the copyright [year](https://github.com/twbs/bootstrap/blob/main/build/banner.js) variable that's used as a template literal doesn't seem to be updating properly since 2023?

* <https://getbootstrap.com/docs/5.3/about/license/>

![image](https://user-images.githubusercontent.com/69180012/219228798-0083886b-3158-455f-a340-0d8f5ad8a04f.png)

* <https://getbootstrap.com/docs/5.3/examples/footers/>

![image](https://user-images.githubusercontent.com/69180012/219228683-02786108-699a-4a99-af1e-f14e57920e3d.png)

There are numerous other files that use the year variable as well, but you all probably know that pretty well already. 🙂

**Comment by julien-deramond** [comment]
v5.3.0-alpha1 was released the 24th of December 2022 and the copyright year (for some files) comes from [site/layouts/shortcodes](https://github.com/twbs/bootstrap/blob/main/site/layouts/shortcodes/year.html). This value is generated [at the build time](https://gohugo.io/functions/now/) if I'm not mistaken, so when we make the release.
Indeed, maybe there's something to do here to rather use a JS call instead of using it at built time, depending on the use case.

Clearly, regarding the examples, could be useful IMO.
However, for some files, it should remain today in 2022: files' headers etc. because it's linked to the release date.
`license.md` is a tricky one because I'm not 100% sure whether it should have the same date as the dist headers files or the current year.

@mdo  and @XhmikosR just for information and thoughts. Maybe we could transform this discussion into an issue (if not already tackled/answered in the past).

**Comment by XhmikosR** [comment]
Nah, it's fine, it's a non-issue to me.
